mobile/examples/Maui/MauiVisionSample/readme.md

```diff
@@ -0,0 +1,105 @@
+# ONNX Runtime MAUI Vision Sample
+
+The [MAUI Vision Sample](MauiVisionSample.sln) demonstrates the use of two different vision models from the [ONNX Model Zoo collection](https://github.com/onnx/models/tree/main), by a [MAUI](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui) app.
+
+## Overview
+The app enables you to take/pick a photo on the device or use a sample image to explore the following models.
+
+### [Mobilenet](https://github.com/onnx/models/tree/main/vision/classification/mobilenet)
+
+Classifies the major object in the image into 1,000 pre-defined classes.
+
+### [Ultraface](https://github.com/onnx/models/tree/f064171f7dd8e962a8a5b34eac8e1bcf83cebbde/vision/body_analysis/ultraface#ultra-lightweight-face-detection-model)
+
+Lightweight face detection model designed for edge computing devices providing detection boxes and scores for a given image.
+
+This model is included in the repository, but has been updated using the onnxruntime python package tools to:
+- remove unused initializers,
+  - `python -m onnxruntime.tools.optimize_onnx_model --opt_level basic <model>.onnx <updated_model>.onnx`
+- make the initializers constant 
```

Comment:
not obvious to me how the initializers are made constant with the command, I thought it updated the ONNX opset to 14?

Reply:
Sorry - I included the wrong command. It was a previous opset update that causes the issue. Updated with correct info to fix the issue.